core/src/main/java/edu/wpi/grip/core/operations/composite/WatershedOperation.java

```diff
@@ -35,11 +53,15 @@
             .initialValueSupplier(ContoursReport::new)
             .build();
 
-    private final SocketHint<Mat> outputHint = SocketHints.Inputs.createMatSocketHint("Output", true);
+    @SuppressWarnings("unchecked")
+    private final SocketHint<ContoursReport> outputHint = new SocketHint.Builder<>(ContoursReport.class)
+            .identifier("Segmented Contours")
```

Comment:
This is a bit too long; it gets cut off and rendered as "`Segmented C...`" on my machine. Maybe just "`Features`"?